schematic/store/synapse.py

```diff
@@ -177,7 +177,8 @@ def getStorageDatasetsInProject(self, projectId: str) -> List[str]:
             foldersTable = self.storageFileviewTable[(self.storageFileviewTable["contentType"] == "dataset") & (self.storageFileviewTable["projectId"] == projectId)]
             areDatasets = True
         else:
-            foldersTable = self.storageFileviewTable[(self.storageFileviewTable["type"] == "folder") & (self.storageFileviewTable["projectId"] == projectId)]
+            #foldersTable = self.storageFileviewTable[(self.storageFileviewTable["type"] == "folder") & (self.storageFileviewTable["projectId"] == projectId)]
+            foldersTable = self.storageFileviewTable[(self.storageFileviewTable["type"] == "folder") & (self.storageFileviewTable["parentId"] == projectId)]
```

Comment:
This is another instance of, "wasn't this the way schematic always worked?" If I'm interpreting this chance correctly, this is now restricting the dataset to only those in the root of a given project ID. I thought this was already the case?

Reply:
true projectId is redundant here - let me change that and re-commit

Reply:
actually looking at the line here and realizing this is in a different function (getStorageDatasetsInProject) and not the manifest retrieval one, scoping by projectId is needed there. 

In the general case, the main/master fileview may span multiple projects and a user might have access to a subset of those; getStorageDatasetsInProject fetches only the datasets in a specific project.